src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_RootScreen.tsx

```diff
@@ -345,7 +352,14 @@ export const Profile_RootScreen = ({navigation}: ProfileProps) => {
               <ThemeText type="heading__component">
                 {t(ProfileTexts.sections.newFeatures.heading)}
               </ThemeText>
-              <BetaTag style={style.betaTag} />
+              <View style={style.betaLabel}>
+                <ThemeText
+                  color="background_accent_3"
+                  style={style.betaLabelText}
+                >
+                  BETA
+                </ThemeText>
+              </View>
```

Comment:
@adriansberg What's the reason for this change 🤔

Reply:
Ehh. Must've snuck in there when rebasing. Fixing as soon as I have time.